docs/content/concepts/assets/asset-observations.mdx

```diff
@@ -108,13 +108,13 @@ def partitioned_dataset_op(context):
 
 ### Observable source assets
 
-<PyObject object="SourceAsset" /> objects may have a user-defined _observation function_
-that returns a <PyObject object="LogicalVersion" />. Whenever the observation
-function is run, an `AssetObservation` will be generated for the source asset
-and tagged with the returned logical version. The logical version is used in
-staleness calculations for downstream assets.
+<PyObject object="SourceAsset" /> objects may have a user-defined <em>
+observation function{" "} </em> that returns a <PyObject object="LogicalVersion" />. Whenever the
```

Comment:
not sure why prettier is doing this

Reply:
It gets cranky when `PyObject` components are the first item in a sentence. If they're preceded by at least one non-space character, it'll leave it alone.

Yuhan and I are aware of the Prettier/Markdown formatting annoyances and will be addressing them in the next few weeks 🎉